Symfony/CS/Tests/Fixer/Symfony/MethodSeparationFixerTest.php

```diff
@@ -204,7 +323,6 @@ public function method444f1()
     }
 
     /**/
-
```

Comment:
Yes, but that is not what is tested there.
The test was `make sure a blank line is added if there is a comment (not a PHPDoc) above the method`, this is now no longer wanted right? So I've updated the test.